io.github.gnu_octave.symbolic.metainfo.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!--
-Copyright 2016, 2019 Colin B. Macdonald
+Copyright 2016, 2019, 2023 Colin B. Macdonald
 
 Copying and distribution of this file, with or without modification,
 are permitted in any medium without royalty provided the copyright
 notice and this notice are preserved.  This file is offered as-is,
 without any warranty.
 -->
 <component type="addon">
-  <id>octave-symbolic</id>
-  <extends>www.octave.org-octave.desktop</extends>
+  <id>io.github.gnu_octave.symbolic.metainfo.xml</id>
```

Comment:
FTR, this underscore is not a typo: hyphens are not recommended by the AppStream docs